src/image-helpers.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @class ImageHelpers
+ */
+export default class ImageHelpers {
+
+  /**
+   * Generates the image src for a resized image with maximum dimensions `maxWidth` and `maxHeight`.
+   * Images do not scale up.
+   *
+   * @method imageForSize
+   * @param {Object} image the original image model to generate the image src for
+   * @param {Integer} maxHeight the maximum height for the image
+   * @param {Integer} maxWidth the maximum width for the image
+   * @return {String} The image src for the resized image
+   */
+  imageForSize(image, maxHeight, maxWidth) {
```

Comment:
Think I'd prefer option args here. Especially because it will be easy to confused the order of height and width.